package.json

```diff
@@ -21,9 +26,9 @@
     "configuration": {
       "title": "FaunaDB",
       "properties": {
-        "faunadb.secretKey": {
+        "faunadb.adminSecretKey": {
           "type": "string",
-          "description": "Reload VSCode after set the secret."
+          "description": "Quit and restart VSCode after you set or change this secret."
```

Comment:
I was wrong about this. "Reload" is sufficient. sorry

Reply:
We are not able to differentiate the key right now. So I should move with  "Reload VSCode after you set or change this secret.". The label was change to say "Admin secret key" instead of "Secret key".

Reply:
<img width="983" alt="Screen Shot 2020-02-12 at 14 00 34" src="https://user-images.githubusercontent.com/3165839/74358448-33f4aa80-4da0-11ea-927a-3ebf4ec78586.png">